sci-mathematics/mathics/mathics-1.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="A free lightweight CAS using Wolfram Language"
+HOMEPAGE="https://mathics.github.io/"
+SRC_URI="https://github.com/mathics/Mathics/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/Mathics-${PV}"
+
+LICENSE="GPLv3"
```

Comment:
This license is invalid, check the travis output! You should further use `repoman full` to check your ebuild and `repoman commit` instead of `git commit`.